### PR TITLE
To match the type of smoothener.add_value

### PR DESCRIPTION
**Pull request by Pendar2**
Loss would always be a single float tensor

### CODE REVIEW

**Comment by jph00** [comment]
Please explain what bug you're fixing, and include the tests requested in the PR template.

**Comment by Pendar2** [comment]
It's not broken as it is right now. Currently the callback.py file is almost completely decoupled from pytorch except in 2 places:
https://github.com/fastai/fastai/blob/master/fastai/callback.py#L221
https://github.com/fastai/fastai/blob/master/fastai/callback.py#L270

This change would allow me to easily use everything in callback.py in the https://github.com/fastai/tf-fit project by adding 2 functions to tensorflow tensors
tf.Tensor.detach = lambda x: x
tf.Tensor.item = lambda x: x.numpy()

**Comment by jph00** [comment]
That's really interesting. Just checking on this - will get back to you asap.

**Comment by Pendar2** [comment]
How about removing .item() on https://github.com/fastai/fastai/blob/master/fastai/callback.py#L270 would that be possible?

**Comment by sgugger** [comment]
That's possible without breaking anything, it that helps you.

**Comment by sgugger** [comment]
I added it in [this commit](https://github.com/fastai/fastai/commit/d5eef11e7f38066ed45439519ca38d9ffeb63179) accidentally. Let me know if I can close this PR.

**Comment by Pendar2** [comment]
That will work, thanks.
